homeassistant/components/zwave_js/api.py

```diff
@@ -106,6 +113,59 @@
 # constants for migration
 DRY_RUN = "dry_run"
 
+# constants for inclusion
+INCLUSION_STRATEGY = "inclusion_strategy"
+PIN = "pin"
+PROVISIONING = "provisioning"
+FORCE_SECURITY = "force_security"
+PLANNED_PROVISIIONING_ENTRY = "planned_provisioning_entry"
+QR_PROVISIONING_INFORMATION = "qr_provisioning_information"
+QR_CODE_STRING = "qr_code_string"
+
+DSK = "dsk"
+
+VERSION = "version"
+GENERIC_DEVICE_CLASS = "generic_device_class"
+SPECIFIC_DEVICE_CLASS = "specific_device_class"
+INSTALLER_ICON_TYPE = "installer_icon_type"
+MANUFACTURER_ID = "manufacturer_id"
+PRODUCT_TYPE = "product_type"
+PRODUCT_ID = "product_id"
+APPLICATION_VERSION = "application_version"
+MAX_INCLUSION_REQUEST_INTERVAL = "max_inclusion_request_interval"
+UUID = "uuid"
+SUPPORTED_PROTOCOLS = "supported_protocols"
+
+UNPROVISION = "unprovision"
+
+# Helper schemas
+PLANNED_PROVISIONING_ENTRY_SCHEMA = vol.Schema(
+    {
+        vol.Required(DSK): str,
+        vol.Required(SECURITY_CLASSES): vol.All(cv.ensure_list, [int]),
```

Comment:
Can we validate the allowed security class values? At least we should know all the possible ones from zwave-js, even if we don't know what the specific device supports, right?